examples/full_stack/main.tf

```diff
@@ -22,14 +22,26 @@ module "vpc_dhcp" {
   #source = "github.com/unifio/terraform-aws-vpc?ref=master//dhcp"
   source = "../../dhcp"
 
-  stack_item_label = "${var.stack_item_label}"
-  stack_item_fullname = "${var.stack_item_fullname}"
-  vpc_id = "${module.vpc_base.vpc_id}"
-  domain_name = "${var.domain_name}"
-  name_servers = "${var.name_servers}"
-  ntp_servers = "${var.ntp_servers}"
+  stack_item_label     = "${var.stack_item_label}"
+  stack_item_fullname  = "${var.stack_item_fullname}"
+  vpc_id               = "${module.vpc_base.vpc_id}"
+  domain_name          = "${var.domain_name}"
+  name_servers         = "${var.name_servers}"
+  ntp_servers          = "${var.ntp_servers}"
   netbios_name_servers = "${var.netbios_name_servers}"
-  netbios_node_type = "${var.netbios_node_type}"
+  netbios_node_type    = "${var.netbios_node_type}"
+}
+
+## Configures ACLs
+resource "aws_network_acl" "acl" {
```

Comment:
@yuhunglin I thought about that, but you have to establish the subnet mappings when declaring the resource. This is the only layer where we have both the VPC ID and the all the subnet IDs.